src/main/java/net/sf/jabref/gui/worker/VersionWorker.java

```diff
@@ -61,22 +63,32 @@ public void done(){
                 return;
             }
 
-            // only respect the ignored version on automated version checks
-            if (latestVersion.equals(toBeIgnored) && !manualExecution) {
-                return;
+            // the newest version, excluding any alpha or beta builds, except if the installed one is one too
+            Version newestVersion = null;
```

Comment:
b/c the `newestVersion` is the new available version which the user gets prompted to download.
If I assign the `installedVersion` it just gets messy.